lib/validator/v1/validator.go

```diff
@@ -411,6 +411,7 @@ func validateICMPFields(v *validator.Validate, structLevel *validator.StructLeve
 func validateRule(v *validator.Validate, structLevel *validator.StructLevel) {
 	rule := structLevel.CurrentStruct.Interface().(api.Rule)
 
+    log.Infof("BCREANE: Inside validateRule: rule=%+v", rule)
```

Comment:
I assume these logs will disappear?

Reply:
gone. glad you noticed them!! not the sort of infamy I'm looking for.